adapters/sharethrough/sharethrough.go

```diff
@@ -35,8 +35,13 @@ type SharethroughAdapter struct {
 func (a SharethroughAdapter) MakeRequests(request *openrtb.BidRequest) ([]*adapters.RequestData, []error) {
 	var reqs []*adapters.RequestData
 
+	if request.Site == nil {
+		return nil, []error{fmt.Errorf("the requests does not have a Site, in App placements are not supported")}
+	}
+	var domain = Util{}.parseDomain(request.Site.Page)
```

Comment:
Just wondering if you're okay sending an empty string as domain because currently the way `parseDomain` works, it can return an empty string

Reply:
Yes, that's expected, thanks for double-checking